### ISSUE TITLE
"Enable jQuery Migrate Helper" settings missing in Calypsofied wp-admin plugin section

### ISSUE THREAD

**Issue by mrfoxtalbot** [Automattic/wp-calypso#48307]
#### Steps to reproduce

1. Install [Enable jQuery Migrate Helper](https://wordpress.org/plugins/enable-jquery-migrate-helper/) plugin
2. Look for its settings.

#### What I expected
I expected to be able to see the plugin's settings in the Calypsofied plugin section, similarly to how the settings are accesible in the regular wp-admin plugins section:

![Screen Shot on 2020-12-14 at 11:18:27](https://user-images.githubusercontent.com/4452464/102081926-5668b180-3e11-11eb-8338-f976e4ab14c1.png)

#### What happened instead
These settings were unavailable:

![Screen Shot on 2020-12-14 at 13:40:19](https://user-images.githubusercontent.com/4452464/102082273-f45c7c00-3e11-11eb-805f-03545ca6c843.png)

#### Browser / OS version
MacOS 0.13.6
Google Chrome  87.0.4280.88

#### Context
19053712-hc

**Comment by mrfoxtalbot** [comment]
I have also added a comment [on the plugin itself](https://wordpress.org/support/topic/add-direct-link-to-plugin-settings/), to suggest we add a direct link to "Settings" on the plugin list page itself (next to the "deactivate" link)

<img width="690" alt="Screen Shot 2020-12-14 at 13 43 40" src="https://user-images.githubusercontent.com/4452464/102082830-e9eeb200-3e12-11eb-887d-c65e52cf7bc6.png">

**Comment by obenland** [comment]
I don't think we'll realistically be able to fix this bug before Nav Unification gets shipped—which will fix this bug. I'm afraid we'll have to live with this a little while longer but it will be a thing of the past soon.
